src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -4,26 +4,36 @@
   },
   "dependencies": {
     "NuProj": "0.10.4-beta-gf7fc34e7d8",
-    "Microsoft.Build.Framework": "0.1.0-preview-00005",
-    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00005",
-    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00005",
+    "Microsoft.Build.Framework": "0.1.0-preview-00015",
```

Comment:
Will do. I already have a branch for that work, but it is still in progress. My branch for that work is https://github.com/joperezr/buildtools/tree/AddBuildToolsPackageDependencies